src/config.rs

```diff
@@ -31,17 +34,21 @@ const CA_ADDRESS: &str = "CA_ADDRESS";
 const TERMINATION_GRACE_PERIOD: &str = "TERMINATION_GRACE_PERIOD";
 const FAKE_CA: &str = "FAKE_CA";
 const ZTUNNEL_WORKER_THREADS: &str = "ZTUNNEL_WORKER_THREADS";
+const PROXY_CONFIG: &str = "PROXY_CONFIG";
 
-const DEFAULT_WORKER_THREADS: usize = 2;
+const DEFAULT_WORKER_THREADS: u16 = 2;
+const DEFAULT_ADMIN_PORT: u16 = 15021;
+const DEFAULT_STATUS_PORT: u16 = 15020;
```

Comment:
Admin was 15000 and status/readiness was 15021 before. Was this incorrectly changed here or is there a corresponding change for the manifests in the Istio repo missing?
/cc @howardjohn @stevenctl

Reply:
Fix: https://github.com/istio/ztunnel/pull/318